### DISCUSSION TITLE
(EN) Transfer of the repositories from Danielperna84 to SukramJ

### DISCUSSION THREAD

**Discussion by SukramJ** [SukramJ/aiohomematic#1930] [Announcements]
Hello,

after Daniel (@danielperna84) has not been actively involved in development for a long time, we have now taken the logical step of transferring the relevant repositories (custom_homematic, hahomematic, pydevccu) to me (@SukramJ).

All existing URLs that previously referred to Daniel's repositories remain valid, as Github redirects them to my repositories.

If you have bookmarks in your browser, it makes sense to change them now (danielperna84 --> sukramj).

The HACS repository is automatically migrated by HACS (Crazy).

So at this point all that remains for me to do is to thank Daniel for the great project idea, the initial implementation and for being a competent contact person. Without this support, I would not have dared to try a CCU integration.
I hope he stays with us for a while with the community work, even if, I have to reveal, he no longer uses Homematic components.

**Comment by dlbogdan** [comment]
I hope you'll keep this project alive and well @SukramJ 
Thank you for everything @SukramJ  and @danielperna84

**Reply by danielperna84** [reply]
You're welcome. 👍
